test/controllers/incomeSources/manage/ManageIncomeSourceControllerSpec.scala

```diff
@@ -71,97 +72,95 @@ class ManageIncomeSourceControllerSpec extends MockAuthenticationPredicate
       ec = ec
     )
 
+  override def beforeEach(): Unit = {
+    disableAllSwitches()
+    enable(IncomeSources)
+  }
+
+  def mockAuthRetrieval(isAgent: Boolean): Unit = {
+    if (isAgent)
+      setupMockAgentAuthRetrievalSuccess(testAgentAuthRetrievalSuccess)
+    else
+      setupMockAuthRetrievalSuccess(testIndividualAuthSuccessWithSaUtrResponse())
+  }
+
+  def getUserSession(isAgent: Boolean): FakeRequest[AnyContentAsEmpty.type] = {
```

Comment:
should be ok to use base type of: AnyContent like

`def getUserSession(isAgent: Boolean): FakeRequest[AnyContent]`